odtp/helpers/filesystem.py

```diff
@@ -0,0 +1,23 @@
+
+import logging
+import shutil
+import os
+import odtp.helpers.settings as config
+
+def create_main_folders(odtp_path = config.ODTP_PATH):
+    components_folder = os.path.join(odtp_path, 'components')
+    users_folder = os.path.join(odtp_path, 'users')
```

Comment:
@caviri a general users folder does not help to separate users: for that there needs to be a folder per user by the github username for example such as 
```
caviri/
sabinem/
...
```

Reply:
Hello @sabinem, sorry I didn't clarify but the `users` folders contains one folder per user, similar to `digital_twins` containing one folder per DT and so. 

```
components/
components/componentA
....
users/
users/sabinem/
users/caviri/
....
users/sabinem/digital_twins/
users/sabinem/digital_twins/digital_twinA
users/sabinem/digital_twins/digital_twinB
...
users/sabinem/digital_twins/digital_twinA/executions/
users/sabinem/digital_twins/digital_twinA/executions/executionsA
```

The reason why not to put directly `digital_twinA` or `digital_twinA` under users is to provide some compartmentalization in case there's another object that the user can have in a future. This allows expanding without the need of changing the path for DTs.

Reply:
@caviri Okay I understand, but then we would also have to adapt the dashboard to always be on the correct working directory. I am not sure what implication that would have on the dashboard. The change seems a bit last minute to me. If you really think it is needed then go ahead with it. 

The PR is not finished yet, correct?